datalake/src/main/java/com/sequenceiq/datalake/controller/sdx/SdxController.java

```diff
@@ -161,4 +169,32 @@ public void retryByCrn(String crn) {
         sdxRetryService.retrySdx(sdxCluster);
     }
 
+    @Override
+    public void start(String name) {
+        String userCrn = threadBasedUserCrnProvider.getUserCrn();
+        SdxCluster sdxCluster = sdxService.getSdxByNameInAccount(userCrn, name);
+        sdxStartService.triggerStart(sdxCluster);
+    }
+
+    @Override
+    public void startByCrn(String crn) {
+        String userCrn = threadBasedUserCrnProvider.getUserCrn();
+        SdxCluster sdxCluster = sdxService.getByCrn(userCrn, crn);
+        sdxStartService.triggerStart(sdxCluster);
+    }
+
+    @Override
+    public void stop(String name) {
+        String userCrn = threadBasedUserCrnProvider.getUserCrn();
+        SdxCluster sdxCluster = sdxService.getSdxByNameInAccount(userCrn, name);
+        sdxStopService.triggerStop(sdxCluster);
+    }
+
+    @Override
+    public void stopByCrn(String crn) {
```

Comment:
I think here we need to do some validation. Only allow stopping of Datalake if there is no Running workload clusters.